en/apps/ios.md

```diff
@@ -0,0 +1,51 @@
+# iOS application
+
+## Purpose of this document
+
+This document describes how you can setup your iOS application to
+work with your wallabag instance. There is no difference in this
+procedure for wallabag v1 or v2.
```

Comment:
Wallabag v1 is not supported

Reply:
Apologies - I used the Android docs as a template. I'll change this

Reply:
No problem, nothing serious. thank you for your work

Reply:
Commit 996b843 should have the change - shown below